package.json

```diff
@@ -9,7 +9,7 @@
     "build": "yarn workspaces foreach -Apt --exclude '{root-workspace}' run build",
     "build:docs": "typedoc --skipErrorChecking --theme default --out docs/api --tsconfig tsconfig.docs.json && touch docs/.nojekyll",
     "bundle": "yarn workspace @cord.network/sdk run bundle",
-    "clean": "rimraf tests/dist && yarn workspaces foreach -p --exclude '{root-workspace}' run clean",
+    "clean": "rimraf tests/dist && yarn workspaces foreach -Ap --exclude '{root-workspace}' run clean",
```

Comment:
this is required in `prepublish`, `publish`, and `publish-beta` options too.

Reply:
True, will send a separate PR for that